src/content/docs/browser/new-relic-browser/browser-pro-features/marks-and-measures.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Automatically track native marks and measures'
+tags:
+  - Browser
+  - Browser monitoring
+  - Experimental features
+metaDescription: "Observes and reports on the performance of your webpages by automatically tracking native marks and measures."
+freshnessValidatedDate: never
+---
+
+<Callout variant="important">
+  This is an experimental browser feature. These features are subject to change and should be used with caution. They are only available when using the browser agent installed via copy/paste or NPM.
+</Callout>
+
+
+[Marks](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark) and [measures](https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure) are standard methods to observe and report on the performance of your webpages. They are generic events native to the browser and can be used to measure the duration of any task. The New Relic browser agent can automatically track marks and measures as store them as `BrowserPerformance` events.
```

Comment:
Suggestions: [Marks](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark) and [measures](https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure) are standard methods to observe and report your webpages' performance. These generic events are native to browser and can measure any task's duration. The New Relic browser agent automatically tracks these marks and measures and then store them as `BrowserPerformance` events.